pkg/operator/encryption/controllers.go

```diff
@@ -18,11 +18,15 @@ import (
 var (
 	apiserverScheme = runtime.NewScheme()
 	apiserverCodecs = serializer.NewCodecFactory(apiserverScheme)
+
+	encoder runtime.Encoder
 )
 
 func init() {
 	utilruntime.Must(apiserverconfigv1.AddToScheme(apiserverScheme))
 	utilruntime.Must(apiserverconfig.AddToScheme(apiserverScheme))
+
+	encoder = apiserverCodecs.LegacyCodec(apiserverconfigv1.SchemeGroupVersion)
```

Comment:
these are not shared traditionally, especially without more specific names. Just construct them locally.